.github/workflows/build.yml

```diff
@@ -129,13 +129,29 @@ jobs:
         uses: actions/checkout@v2
       - name: Prepare tools
         run: |
-          sudo apt-get update && sudo apt-get install -y jq
```

Comment:
We don't need jq anymore?

Reply:
Correct.

We were using it to parse the build matrix file, but that file is now YAML instead of JSON, and we’re parsing it using Python code instead (because apparently you can just use Python code directly in `run:` blocks in GHA if you specify the interpreter properly).